src/modules/vendor/pages/ni/management/CoursesDirectory.vue

```diff
@@ -81,11 +83,20 @@ export default {
     },
     trello () {
       return [
-        { title: 'À venir', courses: this.courses },
-        { title: 'En cours', courses: this.courses },
-        { title: 'Terminée(s)', courses: this.courses },
+        { title: 'À venir', courses: this.courseListForthcoming },
+        { title: 'En cours', courses: this.courseListInProgress },
+        { title: 'Terminée(s)', courses: this.courseListCompleted },
       ]
     },
+    courseListForthcoming () {
+      return this.coursesWithGroupedSlot.filter((course) => this.isForthcoming(course));
```

Comment:
tu peux directement écrire `return this.coursesWithGroupedSlot.filter(this.isForthcoming);` car la définition de ta fonction correspond à la callback de `filter`.